### ISSUE TITLE
Android Unable to merge Dex

### ISSUE THREAD

**Issue by vanlooverenkoen** [jenkinsci/java-client-api#324]
I tried to use your library in my Android App but it does not work. I added (kotlin):

`implementation 'com.offbytwo.jenkins:jenkins-client:0.3.7'`

But I got this error when I wanted to run my app

`Error:Execution failed for task ':app:transformDexArchiveWithExternalLibsDexMergerForDebug'.
> java.lang.RuntimeException: com.android.builder.dexing.DexArchiveMergerException: Unable to merge dex`

**Comment by MaxmaxB** [comment]
I have the same problem did you find any solution ?

**Comment by khmarbaise** [comment]
Can you show a full example how you are using the lib etc....otherwise it's hard to follow what might be wrong...

**Comment by lidudutju** [comment]
It looks like android SDK and the library have the same **UserDataHandler** Class.

>  Task :app:transformDexArchiveWithExternalLibsDexMergerForDebug FAILED
D8: Program type already present: org.w3c.dom.UserDataHandler


FAILURE: Build failed with an exception.
